core/controllers/story_viewer.py

```diff
@@ -217,6 +222,15 @@ def get(self, story_id: str, node_id: str) -> None:
 
     @acl_decorators.can_access_story_viewer_page_as_logged_in_user
     def post(self, story_id: str, node_id: str) -> None:
+        """Records the completion of a specific node within a story.
+
+        Args:
+            story_id: str. The story ID.
+            node_id: str. The node ID.
+
+        Returns:
+            json. A json object.
```

Comment:
This seems wrong.

Reply:
The removing Returns gives the following lint check error.
```
************* Module core.controllers.story_viewer
core/controllers/story_viewer.py:228:4: W9011: Missing return documentation 
core/controllers/story_viewer.py:228:4: W9012: Missing return type documentation 

---------------------------
Linter Checks Failed.
---------------------------

```

Reply:
Hm, if you look at the body of the code, you'll see that there's a "return" at the end. But this is different from all the other handlers, so I think it's an error. Can you remove the "return" before self.return_json at the bottom so that it matches the other handlers?

Reply:
I removed it and it works now.

Reply:
OK, thanks. Feel free to request a review once you've made the changes and replied to all the other comments as well.